content/content-tool/tool/src/webapp/vm/content/chef_resources_deleteConfirm.vm

```diff
@@ -29,7 +29,7 @@
 					<tr>
 						<td headers="title">
 							<h4>
-								<img src = "#imageLink($member.iconLocation)" alt ="$member.label" />
+								<span class="$member.iconClass" title="$!member.label"></span>
```

Comment:
Good work using `span`!

The alt tag is especially useful for accessibility purposes. The W3C does not recommend using the title attribute for accessibility purposes, [source](https://www.w3.org/TR/html/dom.html#the-title-attribute). I *think* the icon just provides visual styling, in which case I would add the `aria-hidden="true"` attribute.

If the icon serves a semantic purpose than the following changes should be made:
- Add <span class="sr-only">$!member.label</span> after the icon.

Adding the `sr-only span`, would correctly capture what was used previously.